Phoenix/PHSpace.m

```diff
@@ -222,9 +237,35 @@ - (void)removeWindows:(NSArray<PHWindow *> *)windows {
                                (__bridge CFArrayRef) @[@(self.identifier)]);
 }
 
+/**
+ * - https://github.com/kasper/phoenix/issues/348
+ * - https://github.com/koekeishiya/yabai/issues/2240#issuecomment-2116326165
+ * - https://github.com/ianyh/Amethyst/issues/1643#issuecomment-2132519682
+ */
+- (void)moveWindowsWithCompatId:(NSArray<PHWindow *> *)windows {
+    NSArray<NSNumber *> *ids = [self identifiersForWindows:windows];
+    NSUInteger windowCount = [ids count];
+    NSMutableData *windowIdSequence = [[NSMutableData alloc] initWithCapacity:windowCount * sizeof(CGWindowID)];
+    for (id item in ids) {
```

Comment:
Type as `NSNumber` instead of `id`.

Reply:
I get the compile error `Interface type cannot be statically allocated`; I think it needs to be `NSNumber *` (which is the type of the array item). Changed to that!